lib/child_process.js

```diff
@@ -161,15 +168,40 @@ function exec(command, options, callback) {
 }
 
 const customPromiseExecFunction = (orig) => {
-  return (...args) => {
+  return (file, options = {}) => {
```

Comment:
Careful here in that the `options = {}` will not catch the case where `options` is explicitly `null`. You likely need to add a condition below on line 179 such that `if (options === null || typeof options !== 'object') {`